peerberrypy/api.py

```diff
@@ -128,19 +128,19 @@ def get_loans(
             self,
             quantity: int,
             start_page: int = 0,
-            max_remaining_term: int = None,
-            min_remaining_term: int = None,
+            max_remaining_term: Optional[int] = None,
+            min_remaining_term: Optional[int] = None,
             max_interest_rate: Optional[decimal.Decimal] = None,
             min_interest_rate: Optional[decimal.Decimal] = None,
             max_available_amount: Optional[decimal.Decimal] = None,
             min_available_amount: Optional[decimal.Decimal] = None,
-            countries: list = None,
-            originators: list = None,
-            loan_types: list = None,
+            countries: Optional[list[str]] = None,
+            originators: Optional[list[str]] = None,
+            loan_types: Optional[list[str]] = None,
```

Comment:
Instead of having `list[str]`, it's better to have `List[str]` to ensure backward compatibility with older Python versions. You can use the List from the typing module.